socorro/signature/rules.py

```diff
@@ -725,11 +725,11 @@ def action(self, raw_crash, processed_crash, notes):
             new_sig = 'IPCError-browser | {}'
         else:
             new_sig = 'IPCError-content | {}'
-        new_sig = new_sig.format(raw_crash['ipc_channel_error'][:100])
+        new_sig = new_sig.format(drop_unicode(raw_crash['ipc_channel_error'][:100]))
```

Comment:
I would do `drop_unicode` and then truncate to 100 characters. That way we have as many ascii characters as possible.

Reply:
Thanks! I will change it.